### PR TITLE
Change instances of Phone to Price

### PR DESCRIPTION
**Pull request by seanlowjk**
Renamed all instance of Phone to price to suit application requirements.

### CODE REVIEW

File: src/main/java/seedu/address/ui/FoodCard.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public FoodCard(Food food, int displayedIndex) {
         this.food = food;
         id.setText(displayedIndex + ". ");
         name.setText(food.getName().fullName);
-        phone.setText(food.getPhone().value);
+        price.setText(food.getPrice().value);
```
Comment (Raikonen) [inline src/main/java/seedu/address/ui/FoodCard.java]: Should add the dollar sign at the start:
price.setText("$" + food.getPrice().value);

File: src/main/java/seedu/address/model/food/Food.java [hunk 2]
```diff
@@ -84,23 +84,23 @@ public boolean equals(Object other) {
...
+        return Objects.hash(name, price, email, tags);
     }
 
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
-                .append(" Phone: ")
-                .append(getPhone())
+                .append(" Price: ")
```
Comment (Raikonen) [inline src/main/java/seedu/address/model/food/Food.java]: Maybe can add dollar sign when displaying price:
.append(" Price: $")

File: src/main/java/seedu/address/model/food/Price.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+/**
+ * Represents a Food's price number in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPrice(String)}
+ */
+public class Price {
+
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Price numbers should only contain numbers, and only in 2 decimal places";
```
[the remark this answers is not in the document]
Comment (seanlowjk) [reply src/main/java/seedu/address/model/food/Price.java]: Have made the changes in [this commit](https://github.com/AY1920S1-CS2103T-F13-2/main/pull/75/commits/c56568a6be094653d109905bb600107cf31a1ac5), thanks!

File: src/main/java/seedu/address/ui/FoodCard.java (hunk 1 above)
Comment (seanlowjk) [reply src/main/java/seedu/address/ui/FoodCard.java]: Have made the changes in [this commit](https://github.com/AY1920S1-CS2103T-F13-2/main/pull/75/commits/c56568a6be094653d109905bb600107cf31a1ac5), thanks!

File: src/main/java/seedu/address/model/food/Food.java (hunk 2 above)
Comment (seanlowjk) [reply src/main/java/seedu/address/model/food/Food.java]: Have made the changes in [this commit](https://github.com/AY1920S1-CS2103T-F13-2/main/pull/75/commits/c56568a6be094653d109905bb600107cf31a1ac5), thanks!

**Pull request merged**